File: example/src/main/java/com/wuxp/codegen/example/evt/QueryOrderEvt.java
package com.wuxp.codegen.example.evt;


import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

import javax.validation.constraints.Max;
import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

@ApiModel("测试的API接口方法一的请求参数")
public class QueryOrderEvt extends BaseQueryEvt {

    @ApiModelProperty(value = "订单sn", example = "test method", required = true)
    @Size(max = 50)
    private String sn;

    @ApiModelProperty(value = "id列表")
    private int[] ids;

}

File: examples/swagger-2/src/test/resources/asserts/typescript/typescript_feign_func/src/evt/QueryOrderEvt.ts
/* tslint:disable */
/* eslint-disable */

        import {BaseQueryEvt} from "./BaseQueryEvt";

    /**
        * 测试的API接口方法一的请求参数
    **/


export interface  QueryOrderEvt extends BaseQueryEvt {

            /**
                *属性说明：订单sn，示例输入：test method
                *sn 约束条件：输入字符串的最小长度为：0，输入字符串的最大长度为：50
                *字段在java中的类型为：String
            **/
        sn: string;
            /**
                *属性说明：id列表，示例输入：
                *字段在java中的类型为：数组
                *字段在java中的类型为：int
            **/
        ids?: number[];
}